.travis.yml

```diff
@@ -1,18 +1,11 @@
 language: python
 python:
-  - "2.7"
   - "3.5"
   - "3.6"
 env:
-  - DJANGO_VERSION=1.11.8
```

Comment:
I think it would be useful to push django version backwards because those old version exists and the developers would be happy to see their old version could use your package without upgrading to 2.0.0. Sometimes, to upgrade to latest version is not possible in an existing project.